class-es-wp-query-wrapper.php

```diff
@@ -685,8 +690,14 @@ public function get_posts() {
 				$allowed_keys[] = 'meta_value';
 				$allowed_keys[] = 'meta_value_num';
 			}
-			$q['orderby'] = urldecode( $q['orderby'] );
-			$q['orderby'] = addslashes_gpc( $q['orderby'] );
+			if ( is_array( $q['orderby'] ) ){
+				$q['orderby'] = array_map( 'urldecode', $q['orderby'] );
+				$q['orderby'] = array_map( 'addslashes_gpc', $q['orderby'] );
+				$q['orderby'] = implode( ' ', $q['orderby'] );
+			} else {
+				$q['orderby'] = urldecode( $q['orderby'] );
+				$q['orderby'] = addslashes_gpc( $q['orderby'] );
+			}
```

Comment:
Orderby needs an overhaul due to recent core updates. I think I'm going to pull this and do it right.